lib/transform.js

```diff
@@ -168,3 +169,23 @@ function mapCheckItemIssueToMkdn(issue) {
   }
   return `- [${checked}] #${issue.number}`;
 }
+
+Transform.usernameToId = function (username) {
+  if (!username) {
+    return;
+  } else {
+    let ownerInfo = memberlist.find((m) =>
+      m.username.includes(username.toLowerCase().slice(1))
+    );
+    return ownerInfo.id;
+  }
+};
+
+Transform.idToUsername = function (id) {
+  if (!id) {
+    return;
+  } else {
+    let Username = memberlist.find((m) => m.id.includes(id)).username;
```

Comment:
Changes these to normal functions. That's just too hard to read. Too many things to parse on one line. Keep it stupid simple. Never clever.